src/ResourceManager/Sql/Commands.Sql/help/New-AzureRmSqlServerVirtualNetworkRule.md

```diff
@@ -0,0 +1,136 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# New-AzureRmSqlServerVirtualNetworkRule
+
+## SYNOPSIS
+Creates an Azure SQL Server Virtual Network Rule.
+
+## SYNTAX
+
+```
+New-AzureRmSqlServerVirtualNetworkRule -VirtualNetworkRuleName <String> -VirtualNetworkSubnetId <String>
+ -ServerName <String> [-ResourceGroupName] <String> [-WhatIf] [-Confirm]
+```
+
+## DESCRIPTION
+This command creates an Azure SQL Server Virtual Network Rule.
```

Comment:
Could you provide more detail describing what a "Azure SQL Server Virtual Network Rule" is or a link to MSDN?

Reply:
We don't have an official MSDN yet. But I will provide more details

Reply:
Odd, I thought I changed this file. Will edit it with the next set of changes.

Reply:
Updated in most recent commit